website/package.json

```diff
@@ -13,9 +13,9 @@
     "build-readme": "./scripts/build-readme.mjs"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.0-alpha.75",
-    "@docusaurus/preset-classic": "2.0.0-alpha.75",
-    "@docusaurus/theme-search-algolia": "^2.0.0-alpha.75",
+    "@docusaurus/core": "2.0.0-beta.15",
+    "@docusaurus/preset-classic": "2.0.0-beta.15",
+    "@docusaurus/theme-search-algolia": "2.0.0-beta.15",
```

Comment:
I forgot we were on an alpha version, I guess alpha->beta is an improvement in stability.

Reply:
The semantic difference between alpha and beta is not _super_ big since we still ship breaking changes. The beta bump was mostly because we finally reached feature parity with v1 after the i18n release, so we are almost there for a stable release